cls/SourceControl/Git/Utils.cls

```diff
@@ -72,6 +72,11 @@ ClassMethod SettingsUIReadOnly() As %Status [ CodeMode = expression ]
 $Get(@..#Storage@("settings","settingsUIReadOnly"), 0)
 }
 
+ClassMethod FavoriteNamespaces() As %String
+{
+    quit $get(@..#Storage@("settings", "user", $username, "favNamespaces"),"")
```

Comment:
Ideally this would be calculated using SQL rather than stored (which as implemented means storing separately in all namespaces even though it's inherently a system-wide setting - not really ideal).